libopenage/cvar/cvar.cpp

```diff
@@ -0,0 +1,39 @@
+// Copyright 2013-2016 the openage authors. See copying.md for legal info.
+
+#include "cvar.h"
+#include "../engine.h"
+
+namespace openage {
+namespace cvar {
+
+bool CVarManager::create(const std::string &name,  const std::pair<get_func, set_func> accessors) {
```

Comment:
the `accessors` param can also be `a const &`, then the move will be used instead of the copy i think.